packages/core/notifications/core-notifications-browser-internal/src/toasts/telemetry/event_types.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { type RootSchema, type EventTypeOpts } from '@kbn/analytics-client';
+
+export enum EventMetric {
+  TOAST_DISMISSED = 'global_toast_list_toast_dismissed',
+}
+
+export enum FieldType {
+  RECURRENCE_COUNT = 'toast_deduplication_count',
+  TOAST_MESSAGE = 'toast_message',
+  TOAST_MESSAGE_TYPE = 'toast_message_type',
+}
+
+const fields: Record<FieldType, RootSchema<Record<string, unknown>>> = {
+  [FieldType.TOAST_MESSAGE]: {
+    [FieldType.TOAST_MESSAGE]: {
+      type: 'keyword',
```

Comment:
Shouldn't this be `text`?

```suggestion
      type: 'text',
```

Reply:
I opted for keyword since there's a finite number of toast titles that can displayed to the user and also I reckon it'd be easier to evaluate the distribution of specific toast titles received through by it's occurrence count this way. let me know what you think.

Reply:
I'm not sure there are finite number of toast titles. The title could have a variable in it, for example, when creating a new data view the title used to be `Created {indexPatterName}`, where `{indexPatterName}` is a variable.